modules/claims_api/app/controllers/claims_api/v2/blueprints/power_of_attorney_request_blueprint.rb

```diff
@@ -4,97 +4,62 @@ module ClaimsApi
   module V2
     module Blueprints
       class PowerOfAttorneyRequestBlueprint < Blueprinter::Base
-        class Veteran < Blueprinter::Base
-          transform Transformers::LowerCamelTransformer
-
-          fields(
-            :first_name,
-            :middle_name,
-            :last_name
-          )
-        end
-
-        class Representative < Blueprinter::Base
-          transform Transformers::LowerCamelTransformer
-
-          fields(
-            :first_name,
-            :last_name,
-            :email
-          )
+        view :index do
+          field :id do |request|
+            request['id']
+          end
+
+          field :type do
+            'power-of-attorney-request'
+          end
+
+          field :attributes do |request|
+            {
+              veteran: {
+                first_name: request['vetFirstName'],
+                last_name: request['vetLastName'],
+                middle_name: request['vetMiddleName']
+              },
+              claimant: {
+                city: request['claimantCity'],
+                country: request['claimantCountry'],
+                military_po: request['claimantMilitaryPO'],
+                military_postal_code: request['claimantMilitaryPostalCode'],
+                state: request['claimantState'],
+                zip: request['claimantZip']
+              },
+              representative: {
+                poa_code: request['poaCode'],
+                vso_user_email: request['VSOUserEmail'],
+                vso_user_first_name: request['VSOUserFirstName'],
+                vso_user_last_name: request['VSOUserLastName']
+              },
+              received_date: request['dateRequestReceived'],
+              decision_date: request['dateRequestActioned'],
```

Comment:
Are we sure we want to return `decisionDate`? Some request statuses are "New," so technically not decided yet. Should this field be `actionedDate` instead?